.github/workflows/integration-tests.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           - rhel-sap
           # TODO(ROX-30765): re-enable once provisioning is fixed
           # - sles
-          - garden-linux
+          # - garden-linux
```

Comment:
Perhaps delete all together.

Reply:
I had the same temptation, but it is interesting to make it easy restore the test locally. Also, deleting the top level entry would mean to delete all the other garden-linux bits (ansible).